specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -0,0 +1,861 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "0.1-preview",
+    "title": "ConfidentialLedgerClient",
+    "description": "The ConfidentialLedgerClient writes and retrieves ledger entries against the Confidential Ledger service."
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{ledgerBaseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "ledgerBaseUrl",
```

Comment:
Can we add this, so that generated clients take the baseUrl once in the constructor rather than on each service method?

`  "x-ms-parameter-location": "client",`